NewsCrawlerTrigger/config/url.js

```diff
@@ -0,0 +1,10 @@
+const url = {
```

Comment:
@shrijan-siris at this point, we should probably move `source` collection from mongodb to a json file here. That will include path for each category too.